docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -26,6 +26,38 @@ This article summarizes the features and functionality you need to be aware of
 before adding new {agent}s or replacing your current {beats} with {agent}s.
 Starting in version 7.14.0, {agent} is generally available (GA).
 
+[discrete]
+[[choosing-between-agent-and-beats]]
+== Choosing between {agent} and {beats}
+
+{agent} is a single binary designed to provide the same functionality that the various {beats} provide today. However, some functionality gaps are being addressed as we strive to reach feature parity.
+
+The following steps will help you determine if {agent} can support your use case:
+
+. Determine if the integrations you need are supported and Generally Available (GA) on Elastic Agent. Integrations that are currently GA are:
++
+* `Auditd`
+* `Elastic Endpoint Security`
+* `Fortinet`
+* `Netflow`
+* `Office365`
+* `Okta`
+* `Palo Alto Networks`
+* `Suricata`
+* `System`
+* `Windows`
+* `Zeek`
++
+NOTE: For more details, refer to {integrations-docs}[{integrations}].
++
+. If the integration is available, check <<supported-outputs-beats-and-agent>> to see whether the required output is also supported.
+
+. Review <<additional-capabilties-beats-and-agent>> to determine if any features required by your deployment are supported. {agent} should support most of the features available on {beats} and is updated for each release.
+
+// QUESTION: would we ever correct a typo in anchor? i.e. https://www.elastic.co/guide/en/fleet/current/beats-agent-comparison.html#additional-capabilties-beats-and-agent
```

Comment:
Yes, especially for anchors that aren't used to derive the html filename. Links to the topic will still work; they just won't send users to the anchor which IMO is OK.

I also think we should fix typos in html filenames, but we'll also need to set up redirects if we do.

Reply:
Good point re: anchors vs. page links.
I'll fix this one then unless any objections.